backend/tests/integration/records/indexer/test_indexer_literature/es_1630825.json

```diff
@@ -1,7 +1,7 @@
ensive observing campaign was launched across the electromagnetic spectrum leading to the discovery of a bright optical transient (SSS17a, now with the IAU identification of AT 2017gfo) in NGC 4993 (at $\\\\sim$40 Mpc) less than 11 hours after the merger by the One-Meter, Two Hemisphere (1M2H) team using the 1 m Swope Telescope. The optical transient was independently detected by multiple teams within an hour. Subsequent observations targeted the object and its environment. Early ultraviolet observations revealed a blue transient that faded within 48 hours. Optical and infrared observations showed a redward evolution over $\\\\sim$10 days. Following early non-detections, X-ray and radio emission were discovered at the transient's position $\\\\sim$9 and $\\\\sim$16 days, respectively, after the merger. Both the X-ray and radio emission likely arise from a physical process that is distinct from the one that generates the UV/optical/near-infrared emission. No ultra-high-energy gamma-rays and no neutrino candidates consistent with the source were found in follow-up searches. (Abridged)\"}], \"acquisition_source\": {\"method\": \"oai\", \"source\": \"arxiv\"}, \"arxiv_eprints\": [{\"categories\": [\"astro-ph.HE\", \"gr-qc\"], \"value\": \"1710.05833\"}], \"control_number\": 1630825, \"imprints\": [{\"date\": \"2017-10-16\"}], \"inspire_categories\": [{\"term\": \"Astrophysics\"}, {\"term\": \"Gravitation and Cosmology\"}], \"keywords\": [{\"source\": \"author\", \"value\": \"gravitational waves\"}, {\"source\": \"author\", \"value\": \"stars: neutron\"}, {\"schema\": \"INSPIRE\", \"value\": \"gamma ray: burst\"}, {\"schema\": \"INSPIRE\", \"value\": \"neutron star: binary\"}, {\"schema\": \"INSPIRE\", \"value\": \"neutron star: binary: coalescence\"}, {\"schema\": \"INSPIRE\", \"value\": \"gamma ray: UHE\"}, {\"schema\": \"INSPIRE\", \"value\": \"optical\"}, {\"schema\": \"INSPIRE\", \"value\": \"gravitational radiation\"}, {\"schema\": \"INSPIRE\", \"value\": \"radio wave\"}, {\"schema\": \"INSPIRE\", \"value\": \"X-ray\"}, {\"schema\": \"INSPIRE\", \"value\": \"electromagnetic\"}, {\"schema\": \"INSPIRE\", \"value\": \"radioactivity\"}, {\"schema\": \"INSPIRE\", \"value\": \"messenger\"}, {\"schema\": \"INSPIRE\", \"value\": \"GLAST\"}, {\"schema\": \"INSPIRE\", \"value\": \"ultraviolet\"}, {\"schema\": \"INSPIRE\", \"value\": \"time delay\"}, {\"schema\": \"INSPIRE\", \"value\": \"infrared\"}, {\"schema\": \"INSPIRE\", \"value\": \"nucleus: decay\"}, {\"schema\": \"INSPIRE\", \"value\": \"VIRGO\"}, {\"schema\": \"INSPIRE\", \"value\": \"star\"}, {\"schema\": \"INSPIRE\", \"value\": \"neutrino\"}, {\"schema\": \"INSPIRE\", \"value\": \"LIGO\"}], \"license\": [{\"imposing\": \"arXiv\", \"url\": \"http://arxiv.org/licenses/nonexclusive-distrib/1.0/\"}, {\"imposing\": \"IOP\", \"license\": \"cc-by\", \"url\": \"http://creativecommons.org/licenses/by/3.0/\"}], \"number_of_pages\": 59, \"preprint_date\": \"2017-10-16\", \"texkeys\": [\"GBM:2017lvd\"]}",
```

Comment:
wasn't this supposed to be sorted.. how does this pass? 😟

Reply:
In `test_indexer_literature.py` we use `json.loads` first so the comparison is made between dictionaries (so they don't need to be sorted).